### PR TITLE
DCS-1022 removed feature flag

### PR DESCRIPTION
**Pull request by GurnankCheema**
The 'manageCourtsEnabled' flag was applied to target the DEV environment, forcing users to select their favourite courts if they hadn't already done so.
This targeting has now been removed as it is a requirement across all environments, not just DEV.

### CODE REVIEW

**Review by mikehalmamoj — DISMISSED** [review]

File: backend/setupAuthorisation.test.js [hunk 1]
```diff
@@ -88,7 +88,7 @@ describe('setupAuthorisation tests', () => {
       nunjucksSetup(app, path)
       const services = /** @type{any} */ ({})
       app.use((req, res, next) => {
-        res.locals = { userRoles: () => [] }
+        res.locals = { userRoles: () => [], preferredCourts: 0 }
```
Comment (andrewrlee) [inline backend/setupAuthorisation.test.js:91]: preferred courts should be an array

File: backend/tests/feedbackAndSupport.test.js [hunk 2]
```diff
@@ -11,6 +11,12 @@ describe('Feedback and support', () => {
     const app = express()
     nunjucksSetup(app, path)
     const services = /** @type{any} */ ({})
+
+    app.use((req, res, next) => {
+      res.locals = { preferredCourts: 0 }
```
Comment (andrewrlee) [inline backend/tests/feedbackAndSupport.test.js:16]: preferred courts should be an array of strings

File: backend/setupAuthorisation.test.js (hunk 1 above)
Comment (GurnankCheema) [reply backend/setupAuthorisation.test.js]: changed to preferredCourts: ['ABRDCT']

File: backend/tests/feedbackAndSupport.test.js (hunk 2 above)
Comment (GurnankCheema) [reply backend/tests/feedbackAndSupport.test.js]: changed to preferredCourts: ['ABRDCT']

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**